### ISSUE TITLE
model cannot donwload and  self.fc1 cannot understand

### ISSUE THREAD

**Issue by 520jefferson** [haocai1992/GPT2-News-Classifier#1]
1, good download model fail ( the network is okay)
2,nn.fc1 = nn.Linear(hidden_size*max_seq_len, num_classes) 
   you can just use the last logits to connect with fc1 , so the logits will has nothing to do with the max_seq_len?

**Comment by 520jefferson** [comment]
and there is a officical clss GPT2ForSequenceClassification, you can use
